.ci/scripts/release-manager.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# This script is executed by the release snapshot stage.
+# It requires the below environment variables:
+# - BRANCH_NAME
+# - VAULT_ADDR
+# - VAULT_ROLE_ID
+# - VAULT_SECRET_ID
+#
+set -uexo pipefail
+
+# set required permissions on artifacts and directory
+chmod -R a+r build/distributions/*
+chmod -R a+w build/distributions
+
+# rename docker files to support the unified release format.
+# TODO: this could be supported by the package system itself
+#       or the unified release process the one to do the transformation
+for i in build/distributions/*linux-arm64.docker.tar.gz*
+do
+    mv "$i" "${i/linux-arm64.docker.tar.gz/docker-image-arm64.tar.gz}"
+done
+
+for i in build/distributions/*linux-amd64.docker.tar.gz*
+do
+    mv "$i" "${i/linux-amd64.docker.tar.gz/docker-image.tar.gz}"
+done
+
```

Comment:
I have not seen where this could be changed in the `apm-server` so, maybe it's done within the Beats repo?

I asked the release platform team whether this could be done within the unified release process, still an open discussion.

Ivan pointed to change this in the generators, though I don't know the impact from the consumers, but we can figure out

Reply:
Pretty certain it's part of Beats's mage stuff. I think `mv`ing it around is fine for now.

Reply:
I just raised https://github.com/elastic/beats/pull/30895 though, it's still in a very early stage... and I've no clue if there is any corner cases that could happen if changing the filename. It's always scary to change the packaging system